packages/components/calendar/__tests__/calendar.spec.tsx

```diff
@@ -2,28 +2,16 @@ import { nextTick } from 'vue'
 import { mount } from '@vue/test-utils'
```

Comment:
Please remove `as any` as possible as you can.
https://github.com/element-plus/element-plus/pull/6682/files#diff-4cff895ccf7da8859ac2d4e5300d00dc9ea5bf00cbc3ef77427e14e8fce60ff1R24